### ISSUE TITLE
Debris interfacing routines, tutorial

### ISSUE THREAD

**Issue by jeffjennings** [discsim/frank#186]
**Is your feature request related to a problem? Please describe -**
Debris routines could be accessed through a terminal fit using fit.py. The model could also have a tutorial.

**Describe the solution you'd like -**
Add debris parameters to parameter file and fit.py. Add jupyter tutorial for debris fit (modestly inclined and edge-on).

**Describe alternatives you've considered -**
N/A

**Additional context -**
N/A

**Comment by rbooth200** [comment]
Feel free - I thought that's what I sent it for. Maybe you should wait for the paper to be out before making it readily available though.

**Comment by jeffjennings** [comment]
Yeah just wanted to make sure. No problem to wait to merge until the paper's out. I can also keep the tutorial local for now.
